src/shipit_uplift/tests/test_coverage.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import json
+import responses
+from shipit_uplift import \
+        coverage_by_changeset_impl, \
+        coverage_summary_by_changeset_impl, \
+        coverage_for_file_impl
+
+
+def test_coverage_supported_extensions_api(client):
```

Comment:
It's fine for now, but in the future we might want to split the tests into multiple files (API endpoint tests in a file, coverage for changeset in another, and so on).